### DISCUSSION TITLE
Sass import ~

### DISCUSSION THREAD

**Discussion by iiLearner** [fwouts/previewjs#1090] [Q&A]
Hi, how would I risolve the import in sass where we import sass libraies using the `~` keyword? e.g:

![image](https://user-images.githubusercontent.com/24250663/195365830-251ee599-f586-4d36-8ce3-c98a2f791439.png)

**Comment by fwouts** [comment]
Hey @iiLearner, it should work without the `~`, would that work for you?

**Reply by iiLearner** [reply]
it wouldn't work without the ~ since as far as I know it's required to include other sass other code from libraries

**Reply by fwouts** [reply]
Would you be able to tell me a bit more about your overall project config, e.g. which framework you use, do you use webpack/vite, which plugins, etc?

If we can find how to make this code work with [Vite](https://vitejs.dev), then we can fix it :)
